AsyncDisplayKit/TextKit/ASTextKitFontSizeAdjuster.mm

```diff
@@ -81,8 +81,8 @@ - (NSUInteger)lineCountForString:(NSAttributedString *)attributedString
   static std::mutex __static_mutex;
   std::lock_guard<std::mutex> l(__static_mutex);
   
-  NSTextStorage *textStorage = [[NSTextStorage alloc] initWithAttributedString:attributedString];
-  NSLayoutManager *layoutManager = _attributes.layoutManagerFactory ? _attributes.layoutManagerFactory() : [[ASLayoutManager alloc] init];
+  NSTextStorage *textStorage = _attributes.textStorageCreationBlock ? _attributes.textStorageCreationBlock(attributedString) : [[NSTextStorage alloc] initWithAttributedString:attributedString];
```

Comment:
If you do any followup, maybe  bump the : down to align with the ? -- these lines are easy to understand but pretty long.